### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1884

### Summary of Changes

* worked on Secondary Panes

### Test Plan

trusting integration tests and an initial page refresh

### CODE REVIEW

File: src/components/SecondaryPanes/CommandBar.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+  getShouldPauseOnExceptions,
+  getShouldIgnoreCaughtExceptions
+} from "../../selectors";
+import Svg from "../shared/Svg";
+import ImPropTypes from "react-immutable-proptypes";
+import { formatKeyShortcut } from "../../utils/text";
+import actions from "../../actions";
+import "./CommandBar.css";
 
 const { Services: { appinfo }} = require("devtools-modules");
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/CommandBar.js]: I punted here because of node unit test mocking

**Comment by clarkbw** [comment]
Do we have an eslint rule for single or double quotes?  I'm seeing a lot of `import React from "react";` and `import React from 'react';` and this might be a good time to make it consistent.

**Comment by jasonLaster** [comment]
hmmm - strange https://github.com/devtools-html/debugger.html/blob/master/.eslintrc#L333-L335

**Comment by clarkbw** [comment]
😕

File: src/components/SecondaryPanes/Breakpoints.js [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import ImPropTypes from "react-immutable-proptypes";
+import classnames from "classnames";
+import actions from "../../actions";
+import { getSource, getPause, getBreakpoints } from "../../selectors";
+import { makeLocationId } from "../../reducers/breakpoints";
+import { truncateStr } from "../../utils/utils";
 const { DOM: dom, PropTypes } = React;
```
Comment (bomsy) [inline src/components/SecondaryPanes/Breakpoints.js]: Any reason this didn't change to 
```
import { DOM as dom, PropTypes } from "react"
```
as in other places?

Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: oops, it should

File: src/components/SecondaryPanes/ChromeScopes.js [hunk 3]
```diff
@@ -139,7 +135,7 @@ const Scopes = React.createClass({
       const loadedProps = this.getObjectProperties(item);
       if (loadedProps) {
         const children = this.makeNodesForProperties(loadedProps, item.path);
-        this.objectCache[actor] = children;
+        this.objectCache[objectId] = children;
```
Comment (clarkbw) [inline src/components/SecondaryPanes/ChromeScopes.js]: Weird that this was working before...

File: src/components/SecondaryPanes/CommandBar.js [hunk 4]
```diff
@@ -1,20 +1,24 @@
-const React = require("react");
-const { DOM: dom, PropTypes } = React;
+import {
+  DOM as dom, PropTypes, createClass
+} from "react";
+
 const { findDOMNode } = require("react-dom");
```
Comment (clarkbw) [inline src/components/SecondaryPanes/CommandBar.js]: Why not `import { findDOMNode } from "react-dom";`?

**Pull request merged**